### DISCUSSION TITLE
[FEAT] About the Specific APPs

### DISCUSSION THREAD

**Discussion by MoMei1583** [eythaann/Seelen-UI#324] [Q&A]
![image](https://github.com/user-attachments/assets/1cb6c824-a3cb-420a-97dd-50ff557265cd)
 **the Specific APPs**
I would like to ask how this feature works and what it does. The settings in it are very complicated and I don't understand what each button means.

**Comment by eythaann** [comment]
well this is like an advanced panel, and here you can add settings for a specific application.

by default some settings are bundled on the seelen-ui project to be out of the box ready to use, and these are read-only. you can add new applications/windows to be target with the identifier an set some specific settings for that app/window.

**Comment by MoMei1583** [comment]
Is there a manual  I can refer to ? Or what information is relevant to
these that I could look up

Eythan David ***@***.***> 于2024年12月7日周六 12:36写道：

**Comment by eythaann** [comment]
ehmm there's no documentation or blog about.
